ui/src/views/Settings.vue

```diff
@@ -19,26 +19,25 @@
       <div class="bx--col-lg-16">
         <cv-tile :light="true">
           <cv-form @submit.prevent="configureModule">
-            <cv-text-input
+            <NsTextInput
               :label="$t('settings.collabora_fqdn')"
-              placeholder="collabora.example.org"
+              :placeholder="$t('settings.placeholder_collabora_domain')"
               v-model.trim="host"
-              class="mg-bottom"
               :invalid-message="$t(error.host)"
               :disabled="loading.getConfiguration || loading.configureModule"
               ref="host"
             >
-            </cv-text-input>
-            <cv-text-input
+            </NsTextInput>
+            <NsTextInput
               :label="$t('settings.collabora_admin_password')"
-              placeholder="my_secret_password"
+              :placeholder="$t('settings.placeholder_admin_password')"
               v-model.trim="admin_password"
```

Comment:
```suggestion
              v-model="admin_password"
```
Let's support passwords with heading or trailing spaces :)